src/prime_rl/trainer/rl/loss.py

```diff
@@ -112,6 +115,7 @@ def compute_loss(
 
         importance_ratio = seq_importance_ratio if loss_config.ratio_type == "sequence" else token_importance_ratio
 
+        advantages = loss_config.adv_tau * advantages + loss_config.teacher_tau * teacher_kl.detach()
```

Comment:
hmmm its not clear to me why we need to have the teacher_kl here.

Why not putting this into the advantages calculation on the orch side ?